README.md

```diff
@@ -27,6 +27,14 @@ metadata:
 ```
 
 ## Running the controller in a cluster
+### Prerequisites
+- go
```

Comment:
go or docker are not really prerequisites for running the controller in a cluster.
Maybe it makes sense to move Prerequisites on one level up.

Reply:
Good point, that makes much more sense! I've updated it here: f2a3664d34a7aa06da995545a87232ca4f344186